hosts/unsigned-int32/home/ashuramaru/home.nix

```diff
@@ -61,13 +61,13 @@
         citra-canary
         mgba
         # dolphin-emu
-        cemu
+        # cemu !!!
         yuzu-mainline
         ryujinx
 
         # Sony
         duckstation
-        pcsx2
+        # pcsx2 TODO!!! remove later
```

Comment:
Why not remove now?

Reply:
misunderstandment, i should have word it correctly, i meant uncomment later, because the package is currently failing to build